src/chat.ts

```diff
@@ -38,9 +38,16 @@ export type ChatRequestData = {
   kind: string;
 };
 
+export type RecipeRequestData = {
+  kind: string;
```

Comment:
**suggestion:** with this interface broken out (nice) I think the `kind` can be a literal, e.g. 
```suggestion
  kind: "recipe_request";
```
for this one, and "user_request" for the other. It'll help avoid accidents I think.

Reply:
great tip thanks!